src/.nuget/Microsoft.NETCore.Runtime.CoreCLR/debian/Microsoft.NETCore.Runtime.CoreCLR.pkgproj

```diff
@@ -23,7 +23,8 @@
     <ArchitectureSpecificNativeFile Include="$(BinDir)mscorlib.ni.dll"/>
     <ArchitectureSpecificLibFile Include="$(BinDir)mscorlib.dll"/>
     <ArchitectureSpecificToolFile Include="$(BinDir)crossgen"/>
-
+    <ArchitectureSpecificToolFile Include="$(BinDir)libclrjit.so"/>
```

Comment:
@schellap In this PR, clrjit.dll is loaded from next to CoreCLR, so unpublished apps should continue to work. When we make the change to place clrjit.dll in its own package, that package will be a dependency of CoreCLR package and thus, should get listed in app.deps.json. I would expect that the host would use that path and pass it to the runtime for it to load the JIT binary. This should work for published (portable and standalone) and unpublished apps.

Am I missing something?